exporters/prometheus/exporter_test.go

```diff
@@ -368,6 +369,26 @@ func TestPrometheusExporter(t *testing.T) {
 				counter.Add(ctx, 9, opt)
 			},
 		},
+		{
+			name:         "with resource attributes filter",
+			expectedFile: "testdata/with_resource_attributes_filter.txt",
+			options: []Option{
+				WithResourceAsConstantLabels(&filter),
+			},
```

Comment:
We probably want a testcase where some of the resources attributes are written to the metric.

Currently we use a resource built [here](https://github.com/codeboten/opentelemetry-go/blob/eee76e98a6348979a75db4fa9331989bcc10ada5/exporters/prometheus/exporter_test.go#L405-L412) with:
```
ServiceName: prometheus_test
TelemetrySDKVersion: latest
```
and anything put in the `customResouceAttrs` test case field.

Reply:
added a test for including only the service name